pages/windows/eventcreate.md

```diff
@@ -4,7 +4,7 @@
 > Event IDs can be any number between 1 and 1000.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/eventcreate>.
 
-- Create a new event with a given id (1-1000) in the log:
+- Create a new event with a given ID (1-1000) in the log:
```

Comment:
?
```suggestion
- Create a new event with a given ID (1 to 1000 inclusive) in the log:
```

Reply:
I've always been a bit of a fan of the Rust way of denoting this, but I guess based on your Linux comment it might not be favorable to use here. 🤔 

I believe they do `..` for exclusive range (1..1000), and `..=` for inclusive range (1..=1000).

I think it's important to denote if it's inclusive/exclusive, but wondering if there's really any need to replace the "-" with "to".
Any other opinions?

---

Something we should probably think about at some point is the localization of numbers. 🤔
There are some places (though not many) that we use thousand separators in numbers.

Consider the following: 3,141

To someone in France, does this mean 3.141 (three point one four one), or 3,141 (three thousand one hundred and forty-one).

As it's not localized, there's simply no way to tell without context or knowing if we localize strings or not.

Reply:
Oh wait, I'm an idiot. -_-
Ignore my comment about localizing numbers, I guess that can be handled in said translations as is.

Sorry, I was thinking about how like in software we need to specific set a locale in because numbers can be calculated. I guess that doesn't matter in this case since all numbers are essentially static strings, and can be translated as such.

Reply:
>To someone in France, does this mean 3.141 (three point one four one), or 3,141 (three thousand one hundred and forty-one).

Good point.  Removing the separator altogether seems like the most expedient solution: 3141

Reply:
>I've always been a bit of a fan of the Rust way of denoting this, but I guess based on your Linux comment it might not be favorable to use here. 🤔 
>
>I believe they do `..` for exclusive range (1..1000), and `..=` for inclusive range (1..=1000).

I agree, Rust's terseness for numeric ranges awesome, but...

>When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness.

Reply:
> Good point. Removing the separator altogether seems like the most expedient solution: 3141

I remember seeing a localized number before, so I assumed we had many. Just checked, but it seems we only have the one:

> Benchmark a database with 10 clients, 2 worker threads, and 10,000 transactions per client:
> \- phbench.md

And this one doesn't suffer from the ambiguity described above, so I guess it's not a big deal.

---

Think we should include something about this in the style guide anyway?

Regardless, I believe we need to ensure all numbers have no localization.

From there, a decision can be made:
* Leave them delocalized for all users and that's it.
* Use a syntax to denote localizable numbers, tldr clients would be responsible for if they want to apply the system locale or not, or may choose to make it a setting for users.

I'm a fan of localized numbers myself, as I find them easier to read. It depends on what everyone else thinks, though.

Reply:
Following [a conversation on Gitter](https://gitter.im/tldr-pages/tldr?at=60cf826b258ebf6d92dced1e), we established that locales are not the same as language. So, we have 2 options:

1. Localise numbers to the language of the translation (but someone can have a localisation setting that isn't the same as the language they are viewing the page in
2. Avoid localising numbers at all

Reply:
I don't really understand why the client has something to do with this. If I have an English page I write `2021-06-23` and if I translate that to German, I just write `23.06.2021`. Or am I misunderstanding something?

Reply:
Language and locale are different things, more info at https://github.com/tldr-pages/tldr/issues/6147